lib/zendesk/client/operation.ex

```diff
@@ -5,15 +5,16 @@ defmodule Zendesk.Client.Operation do
   alias Zendesk.Client.Parser
   alias __MODULE__
 
-  defstruct path: "/", type: :get, parser: &Parser.default_parse/1, params: []
+  defstruct path: "/", type: :get, parser: &Parser.default_parse/1, params: [], body: %{}
```

Comment:
What do you think about having the body be default `nil`?  It's not used in many API calls, and at some point in the future it might be nice in the `get` calls to make sure the body is empty (`nil`).  🤷‍♂️ 

```suggestion
  defstruct path: "/", type: :get, parser: &Parser.default_parse/1, params: [], body: nil
```